backend/techpick-ranking/src/main/java/techpick/ranking/domain/pick/PickRankingService.java

```diff
@@ -35,34 +36,53 @@ public class PickRankingService {
 	 *     Ex. 일별 집계, 월별 집계, 연별 집계 테이블을 나눠서 미리 연산.
 	 *         API 호출은 일별, 월별, 연별을 나눠서 호출하도록 변경하면 해결 가능.
 	 */
+	@Cacheable(cacheNames = "daily_link_rank")
```

Comment:
p3: 이 부분은 하드코딩보다 CachType enum을 이용하면 어떨까요! 

```suggestion
	@Cacheable(cacheNames = CacheType.DAILY_LINK_RANK.name())
```

Reply:
어노테이션에 `CacheType.DAILY_LINK_RANK.getCacheName()`으로 값을 가져올 수 없네요..
메서드 내부에서는 호출이 가능한 것으로 보아 어노테이션이 컴파일에서 동작하는 것이 아닌 런타임에서 동작하기 때문이지 않을까요?
그래서 컴파일러가 인식하지 못하는 것 같아요

Reply:
`CacheType`에 `public static final String DAYILY_LINK_RANK = "daily_link_rank"`
이런식으로 만들어 두면 쓸 수 있지 않을까요? [(ref)](https://stir.tistory.com/321)